src/api/app/views/webui/users/notifications/_notification_filter.html.haml

```diff
@@ -70,6 +70,11 @@
                                                               key: 'kind[appealed_decisions]', name: 'kind[]', value: 'appealed_decisions',
                                                               label_icon: 'fas fa-hand',
                                                               checked: selected_filter[:kind]&.include?('appealed_decisions') }
+    .px-4.py-1
+      = render partial: 'webui/shared/check_box', locals: { label: 'Member on Groups', amount: counted_notifications['member_on_groups'],
```

Comment:
```suggestion
      = render partial: 'webui/shared/check_box', locals: { label: 'Group', amount: counted_notifications['member_on_groups'],
```

Reply:
I didn't want to use `Group(s)` because there is a `Groups` section in the filter and could be confusing.

![Screenshot(1)](https://github.com/openSUSE/open-build-service/assets/2581944/5377bf79-cf3e-40aa-a60b-2642aa810592)


I'll use `Groups membership` if you don't mind.

Reply:
Done